modules/vault_cluster/variables.tf

```diff
@@ -65,4 +65,8 @@ variable "lambda_secondary_subnet_cidr" {
 
 variable "lb_subnets" {
   description = "Subnets to place LB in"
+}
+
+variable "instance_subnets" {
+  description = "Subnets to place Vault Instances in"
```

Comment:
Should we include the `type` (like `string` or `list[string]`)? Forgive the pseudo code here.

Reply:
Added this, hadn't originally due to the others not including it. However, I did in the variables file in cn-vault as part of https://github.com/datarobot/cn-vault/pull/6